inspirehep/modules/disambiguation/config.py

```diff
@@ -25,16 +25,19 @@
 from __future__ import absolute_import, division, print_function
 
 
-DISAMBIGUATION_CLUSTERS_PATH = '/tmp/clusters.json'
+DISAMBIGUATION_BASE_PATH = '/var/data/disambiguation/'
```

Comment:
This will break during deployment, unless properly configured (overridden) or setup properly in puppet/similar. Not that this pr should do anything about it, just pointing it out.

Reply:
It should break in docker too imo :/ (test user should not be able to create a dir there by itself).

Reply:
What I meant to do here is to use `var/data/disambiguation` like `invenio-records-files` uses `var/data/records` or `invenio-workflows-files` uses `var/data/workflows`. As for why this doesn't fail in the tests, I mock everything with `tmpdir`.